lib/git_worktree.rb

```diff
@@ -359,9 +377,9 @@ def open_repo
     @repo = Rugged::Repository.new(@path)
   end
 
-  def clone(url)
+  def clone(url, bare = true)
```

Comment:
Note:  Decided to toss in a default value of `true` here just to retain the previous functionality in case this happened to be called directly elsewhere, though I don't think it ever was.